src/app/space/create/deployments/apps/deployment-card.component.spec.ts

```diff
@@ -73,67 +82,158 @@ class FakeDeploymentDetailsComponent {
   @Input() spaceId: string;
 }
 
-describe('DeploymentCardComponent', () => {
+function initMockSvc(mockSvc: jasmine.SpyObj<DeploymentsService>, active: Subject<boolean>) {
+  mockSvc.getVersion.and.returnValue(Observable.of('1.2.3'));
+  mockSvc.getCpuStat.and.returnValue(Observable.of({ used: 1, quota: 2 }));
+  mockSvc.getMemoryStat.and.returnValue(Observable.of({ used: 3, quota: 4, units: 'GB' }));
+  mockSvc.getAppUrl.and.returnValue(Observable.of('mockAppUrl'));
+  mockSvc.getConsoleUrl.and.returnValue(Observable.of('mockConsoleUrl'));
+  mockSvc.getLogsUrl.and.returnValue(Observable.of('mockLogsUrl'));
+  mockSvc.deleteApplication.and.returnValue(Observable.of('mockDeletedMessage'));
+  mockSvc.isApplicationDeployedInEnvironment.and.returnValue(active);
+}
```

Comment:
Rather than creating a "blank" service and having this function configure out, how about having this function both create and configure (and return) the mock? Also, instead of passing the "active" value in, the callsite can just take the result and do `mockSvc.isApplicationDeployedInEnvironment.and.returnValue(active)` on their own, since they already have their own `active` value.